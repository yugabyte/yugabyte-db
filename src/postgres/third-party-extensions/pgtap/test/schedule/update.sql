\unset ECHO
\i test/psql.sql

CREATE EXTENSION pgtap VERSION :'old_ver';
SELECT plan(2);
SELECT is(
    (SELECT extversion FROM pg_extension WHERE extname = 'pgtap')
    , :'old_ver'
    , 'Old version of pgtap correctly installed'
);

ALTER EXTENSION pgtap UPDATE TO :'new_ver';

SELECT is(
    (SELECT extversion FROM pg_extension WHERE extname = 'pgtap')
    , :'new_ver'
    , 'pgtap correctly updated'
);

SELECT finish();
