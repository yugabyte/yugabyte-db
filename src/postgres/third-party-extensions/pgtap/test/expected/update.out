\unset ECHO
1..2
ok 1 - Old version of pgtap correctly installed
ok 2 - pgtap correctly updated
